ragna/deploy/_ui/styles.py

```diff
@@ -11,109 +11,69 @@ def divider():
 
 
 def apply_design_modifiers():
+    apply_design_modifiers_global()
     apply_design_modifiers_source_accordion()
-    # add here calls to other design modifiers,
-    #   group them per UI component
+    apply_design_modifiers_auth_page()
+    apply_design_modifiers_central_view()
+    apply_design_modifiers_chat_info()
+    apply_design_modifiers_chat_interface()
+
+
+def add_modifier(
+    modifier_class: Type[Any], modifications: Any, property: str = "stylesheets"
+):
+    if modifier_class not in pn.theme.fast.Fast.modifiers:
+        pn.theme.fast.Fast.modifiers[modifier_class] = {}
+
+    if property not in pn.theme.fast.Fast.modifiers[modifier_class]:
+        pn.theme.fast.Fast.modifiers[modifier_class] = {property: [modifications]}
+    else:
+        pn.theme.fast.Fast.modifiers[modifier_class][property].append(modifications)
+
+
+def apply_design_modifiers_global():
+    add_modifier(
+        pn.widgets.TextInput,
+        """ .bk-input {border-color: var(--neutral-color) !important;} """,
```

Comment:
CSS file or helper function.

Reply:
will move to CSS file as indicated here https://github.com/Quansight/ragna/pull/312#discussion_r1481478256